docs/install/applying-administrator-updates.md

```diff
@@ -38,35 +38,53 @@ There are three types of administrator updates to Visual Studio:
 
 The title of each administrator update describes both the applicable version range and the resultant version of the update. For example,
 
-* **Visual Studio 2019 version 16.7.0 to 16.7.12 update** classified as a “Security Update” will apply to any Visual Studio edition on the client between versions 16.7.0 through 16.7.12, and it will update those client editions to 16.7.12.  
+::: moniker range="vs-2017"
 
-* **Visual Studio 2019 version 16.0.0 to 16.9.0 update** classified as a “Feature Pack” will apply to select Visual Studio editions on the client between the entire product version range of 16.0.0 through 16.9.0, and it will update those client editions (that haven’t been configured to remain on an earlier servicing baseline) to 16.9.0. 
+* **Visual Studio 2019 version 15.9.0 to 15.9.35 update** classified as a “Security Update” will apply to any Visual Studio 2017 edition on the client between versions 15.9.0 through 15.9.35, and it will update those client editions to 15.9.35.
 
-* **Visual Studio 2019 version 16.8.0 to 16.8.7 update** classified as simply “Updates” will apply to select Visual Studio editions on the client between versions 16.8.0 through 16.8.7, and it will update those client editions to 16.8.7. 
+* **Visual Studio 2019 version 15.0.0 to 15.9.0 update** classified as a “Feature Pack” will apply to Visual Studio 2017 editions licensed for enterprise use on the client between the entire product version range of 15.0.0 through 15.9.0, and it will update those client editions to 15.9.0. Applying this feature pack basically enables the clients to then receive security updates. 
```

Comment:
```suggestion
* **Visual Studio 2017 version 15.0.0 to 15.9.0 update** classified as a “Feature Pack” will apply to Visual Studio 2017 editions licensed for enterprise use on the client between the entire product version range of 15.0.0 through 15.9.0, and it will update those client editions to 15.9.0. Applying this feature pack basically enables the clients to then receive security updates. 
```